dns/cloudflare.go

```diff
@@ -106,10 +108,18 @@ func (cf *Cloudflare) addUpdateDomainRecords(recordType string) {
 			return
 		}
 
-		// 存在参数才进行筛选
-		comment := domain.GetCustomParams().Get("comment")
-		if comment != "" {
-			comment = fmt.Sprintf("&comment=%s", comment)
+		// The name of DNS records in Cloudflare API expects Punycode.
+		//
+		// See: cloudflare/cloudflare-go#688
+		domain.ASCII()
+
+		params := url.Values{}
+		params.Set("type", recordType)
+		params.Set("name", domain.String())
+		params.Set("per_page", pageSize)
```

Comment:
`pageSize` -> 50

Reply:
This variable is added because the magic number is used twice, but feel free to remove it.

Reply:
you can remove it. just like `params.Set("status", "active")`

Reply:
> you can remove it. just like `params.Set("status", "active")`

Do you mean removing the `per_page` parameter, or the `pageSize` variable?

Reply:
remove the `pageSize` variable, just like `params.Set("status", "active")`

Reply:
> remove the `pageSize` variable, just like `params.Set("status", "active")`

Got it. Fixed.